dockstore-webservice/src/test/resources/testParameterFile1.json

```diff
@@ -0,0 +1,70 @@
+{
+  "TopMedVariantCaller.input_cram_files": ["https://storage.googleapis.com/topmed_workflow_testing/topmed_aligner/input_files/NWD176325.recab.cram"],
```

Comment:
I am a bit concerned about using this Google bucket for testing. The bucket was setup several years ago under a UCSC account for the COMMONS grant with Requestor Pays turned off to allow grant participants access to test files. The Requestor Pays option could be turned on at any time or the bucket might simply be deleted. I think we ought to have our own storage location for our test files.